docs/generated/packages/gradle/documents/overview.md

```diff
@@ -45,16 +74,59 @@ nx.bat build <your gradle library>
 
 In any Nx workspace, you can install `@nx/gradle` by running the following command:
 
+{% tabs %}
+{% tab label="Mac/Linux" %}
+
+```shell {% skipRescope=true %}
+./nx add @nx/gradle
+```
+
+{% /tab %}
+{% tab label="Windows" %}
+
 ```shell {% skipRescope=true %}
-nx add @nx/gradle
+nx.bat add @nx/gradle
 ```
 
+{% /tab %}
+{% /tabs %}
+
 ## How @nx/gradle Infers Tasks
 
-The `@nx/gradle` plugin will create an Nx project for each Gradle configuration file present. Any of the following files will be recognized as a Gradle configuration file:
+The `@nx/gradle` creates Nx projects by utilizing a brand new Gradle plugin: `dev.nx.gradle.project-graph`.
+When using `nx add`, the plugin is added as a dependency to root project Gradle file.
+In most cases, the generator should add the task definition to trigger the plugin but if it's missing, add the following configuration to your Gradle configuration:
 
-- `build.gradle`
-- `build.gradle.kts`
+```groovy {% fileName="build.gradle" %}
+plugins {
+    id("dev.nx.gradle.project-graph") version("+")
+}
+
+allprojects {
+    apply {
+        plugin("dev.nx.gradle.project-graph")
+    }
+}
+```
+
+The Gradle plugin creates the nodes and available tasks for each node into a JSON output that Nx can understand. This is entirely based on the Gradle configuration.
+If you ever need to rebuild the graph output, run the following command:
```

Comment:
could be my lack of knowledge here but why would people need to invoke gradlew and rebuild the output? seems like the same as "us nx reset" but that's just bandaid-ing over the issue that we should instead look at fixing. 

Maybe moving this advice into a smaller troubleshooting section at the end for people instead of highlighting is making it seem like a 'normal' operation with the plugin.